kpconv_torch/test.py

```diff
@@ -135,19 +142,24 @@ def main(args):
             config,
             split="validation" if args.filename is None else "test",
             use_potentials=True,
-            infered_file=args.filename,
```

Comment:
passé en attribut de la classe Config

Reply:
I think it will lead to a counterintuitive design (see my previous remarks about the meaning of the configuration, strongly related to training sessions).

Reply:
OK, see above.